src/api/services/ingestion.go

```diff
@@ -626,6 +602,9 @@ func (i *IngestionService) ProcessVcf(
 					if hasGenotype && k == genotypePosition {
 						// create genotype from value
 						gtString := tmpValueStrings[k]
+
+						// TODO: check and handle when 'gtString' is '.'
```

Comment:
this is quite likely to come up... even if you just skip it for now that should be ok